engines/group/app/models/group/group.rb

```diff
@@ -6,11 +6,11 @@ class Group < ActiveRecord::Base
     has_many :memberships, dependent: :destroy
     has_many :users, through: :memberships
     has_many :admins,
-      -> { where "memberships.role = #{ Membership::ROLES[:admin] }" },
+      -> { where "memberships.role = #{Role.new(:admin).index}" },
     through: :memberships,
       source: :user
     has_many :waiters,
-      -> { where "memberships.role = #{ Membership::ROLES[:waiting] }" },
+      -> { where "memberships.role = #{Role.new(:waiters).index}" },
```

Comment:
Well, the issue is that with invitations you still don't have users created. Meanwhile if you accept an invitation to a group and the group has a waiting list you need to wait. And your invitation has long gone